doc/tutorial/climada_engine_Impact.ipynb

```diff
@@ -137,7 +137,7 @@
     "Set first the `Exposures` and use its coordinates information to set a matching `Hazard`.\n",
     "\n",
     "Hint:\n",
-    "The configurable parameter `max_matrix_size` defined in the __[configuration file](../../climada/conf/defaults.conf)__ (located at `/climada/conf/defaults.conf`) controls the maximum matrix size contained in a chunk. You can decrease its value if you are having memory issues when using the `Impact.calc()` method. A high value will make the computation fast, but increase the memory use. "
+    "The configurable parameter `max_matrix_size` defined in the __[configuration file](https://github.com/CLIMADA-project/climada_python/blob/main/climada/conf/climada.conf)__ (located at `/climada/conf/climada.conf`) controls the maximum matrix size contained in a chunk. You can decrease its value if you are having memory issues when using the `Impact.calc()` method. A high value will make the computation fast, but increase the memory use. "
```

Comment:
```suggestion
    "The configurable parameter `max_matrix_size` controls the maximum matrix size contained in a chunk. You can decrease its value if you are having memory issues when using the `Impact.calc()` method.\n",
    "A high value will make the computation fast, but increase the memory use.\n",
    "(See [Configuration]() on how to set configuration values.)"
```

Reply:
Thanks a lot! But I couldn't commit it either... "This diff has recently been updated." again and refreshing doesn't help.